src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java

```diff
@@ -13,7 +12,6 @@
 public class StatsDOutputWriterIntegrationTest {
 
     @Test
-    @Ignore
     public void test() throws IOException {
```

Comment:
Can we add non number metric value test case and make sure we don't write to StatsDOutuputWriter